framework/base/Module.php

```diff
@@ -705,4 +705,60 @@ public function afterAction($action, $result)
         $this->trigger(self::EVENT_AFTER_ACTION, $event);
         return $event->result;
     }
+
+    /**
+     * Layout find in parent module.
+     * @return array [module, layout]
+     */
+    private function findLayout()
```

Comment:
additionally, it returns the $module + $layout, so the method should match this return result